cif_pow.dic

```diff
@@ -3326,150 +3387,6 @@ save_pd_calib_std.external_name
-       _pd_calib_wavelength.diffractogram_id   DIFFRACTOGRAM_A
-       _pd_calib_wavelength.phase_id           ACME_CORUNDUM
-;
-;
-       The incident wavelength used in the measurement identified by the
-       _diffrn.id 'DIFFRN_EXP_A' was refined from a diffractogram with a
-       _pd_diffractogram.id value of 'DIFFRACTOGRAM_A'. In this
-       diffractogram, a phase with a _pd_phase.id value of 'ACME_CORUNDUM'
-       was used to calibrate the wavelength against known unit cell
-       parameters.
-;
-
-save_
-
-save_pd_calib_wavelength.diffractogram_id
-
-    _definition.id                '_pd_calib_wavelength.diffractogram_id'
-    _definition.update            2023-01-17
-    _description.text
-;
-    A code which identifies a diffractogram which was used in the calibration of
-    the wavelength.
-;
-    _name.category_id             pd_calib_wavelength
-    _name.object_id               diffractogram_id
-    _name.linked_item_id          '_pd_diffractogram.id'
-    _type.purpose                 Link
-    _type.source                  Assigned
-    _type.container               Single
-    _type.contents                Text
-
-save_
-
-save_pd_calib_wavelength.diffrn_id
-
-    _definition.id                '_pd_calib_wavelength.diffrn_id'
-    _definition.update            2023-01-17
-    _description.text
-;
-    A code which identifies the diffraction experiment to which this calibration
-    belongs.
-;
-    _name.category_id             pd_calib_wavelength
-    _name.object_id               diffrn_id
-    _name.linked_item_id          '_diffrn.id'
-    _type.purpose                 Link
-    _type.source                  Assigned
-    _type.container               Single
-    _type.contents                Word
-
-save_
-
-save_pd_calib_wavelength.phase_id
-
-    _definition.id                '_pd_calib_wavelength.phase_id'
-    _definition.update            2023-01-17
-    _description.text
-;
-    A code which identifies a phase whose cell parameters were used in the
-    calibration of the wavelength.
-;
-    _name.category_id             pd_calib_wavelength
-    _name.object_id               phase_id
-    _name.linked_item_id          '_pd_phase.id'
-    _type.purpose                 Link
-    _type.source                  Assigned
-    _type.container               Single
-    _type.contents                Text
-
-save_
-
-save_pd_calib_wavelength.special_details
-
-    _definition.id                '_pd_calib_wavelength.special_details'
-    _definition.update            2023-01-17
-    _description.text
-;
-    Description of intensity calibration details that cannot otherwise be
-    recorded using other PD_CALIB_WAVELENGTH data items
-;
-    _name.category_id             pd_calib_wavelength
-    _name.object_id               special_details
-    _type.purpose                 Describe
-    _type.source                  Recorded
-    _type.container               Single
-    _type.contents                Text
-
```

Comment:
We are losing this definition, which I'm not too troubled about, but thought I should note it. It could also be added as an extra loop column in the same way as the other two definitions if that is thought worthwhile.

Reply:
can include a special_details data name with the others. Not any extra work.